src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -578,20 +604,22 @@ private Mqtt5Client createCrtClient() throws MessageClientException {
         TlsContext tlsContext = null;
         TlsContextOptions tlsContextOptions = null;
         try {
-            Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder builder
-                    = new Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder(brokerUri.getHost(), port)
+            Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder builder =
+                    new Mqtt5ClientOptions.Mqtt5ClientOptionsBuilder(brokerUri.getHost(), port)
                     .withLifecycleEvents(connectionEventCallback)
                     .withPublishEvents(publishEventsCallback)
                     .withSessionBehavior(Mqtt5ClientOptions.ClientSessionBehavior.REJOIN_POST_SUCCESS)
-                    .withOfflineQueueBehavior(
-                            Mqtt5ClientOptions.ClientOfflineQueueBehavior.FAIL_ALL_ON_DISCONNECT)
+                    .withOfflineQueueBehavior(Mqtt5ClientOptions.ClientOfflineQueueBehavior
+                            .FAIL_ALL_ON_DISCONNECT)
                     .withConnectOptions(new ConnectPacket.ConnectPacketBuilder()
                             .withRequestProblemInformation(true)
-                            .withClientId(clientId).build())
-                    // TODO configurable?
+                            .withClientId(clientId)
+                            .withSessionExpiryIntervalSeconds(sessionExpiryInterval)
+                            .withMaximumPacketSizeBytes(maximumPacketSize != null ? maximumPacketSize :
```

Comment:
withMaximumPacketSizeBytes can accept null which will use the default. Do not provide our own default.

Reply:
I received a null pointer error when I did that earlier, might've stemmed from something else so I'll revisit this